tests/test_extract_coding.py

```diff
@@ -129,14 +135,48 @@ def true_protein_coding_fasta_string(true_protein_coding_fasta_path):
         return f.read()
 
 
+@pytest.fixture
+def jaccard_threshold(molecule):
+    from khtools.extract_coding import get_jaccard_threshold
+    threshold = get_jaccard_threshold(None, molecule)
+    return threshold
+
+
+@pytest.fixture
+def peptide_ksize(molecule):
+    from khtools.bloom_filter import get_peptide_ksize
+
+    ksize = get_peptide_ksize(molecule)
+    return ksize
+
+
+@pytest.fixture
+def single_alphabet_ksize_true_scores_path(data_folder):
+    true_scores_path = os.path.join(
+        data_folder, "extract_coding",
+        "SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22__"
+        "molecule-protein_ksize-7.csv")
+    return 'protein', 7, true_scores_path
```

Comment:
What do protein and 7 mean here? I think having a named tuple would make things more clear and then you wouldn’t have to unpack each time you used the fixture